lib/python/Screens/InfoBarGenerics.py

```diff
@@ -3913,7 +3913,7 @@ def startInstantRecording(self, limitEvent=False):
 				end = info["end"]
 		else:
 			if limitEvent:
-				self.session.open(MessageBox, _("No event info found, recording indefinitely."), MessageBox.TYPE_INFO)
+				self.session.open(MessageBox, _("No event info found, recording default is indefinitely."), MessageBox.TYPE_INFO)
```

Comment:
"Recording indefinitely" is grammatically correct here, albeit functionally inaccurate, as "indefinite" in this case actually meeans (from what I can recall) 24 hours.